os/board/rtl8730e/src/component/mbed/targets/hal/rtl8730e/flash_api.c

```diff
@@ -275,15 +273,13 @@ int  flash_write_word(flash_t *obj, u32 address, u32 data)
   * @param  len: Length of the data to read.
   * @param  data: Address to save the readback data.
   * @return Read result.
-  * @retval 1: Success.
-  * @retval Others: Error.
+  * @retval len: Total length of bytes read.
   * @note It is ok to read under auto mode due to flash cache.
   */
 int  flash_stream_read(flash_t *obj, u32 address, u32 len, u8 *data)
 {
 	/* To avoid gcc warnings */
 	(void) obj;
-	FLASH_Write_Lock();
```

Comment:
can we add the zero length check at the beginning of the function?

Reply:
Yes sure, I have added, can you help to check again~?